doc/source/serve/monitoring.md

```diff
@@ -128,17 +145,17 @@ clients:
   - url: http://localhost:3100/loki/api/v1/push
 
 scrape_configs:
-- job_name: ray
-static_configs:
-  - labels:
-    job: ray
-    __path__: /tmp/ray/session_latest/logs/serve/*.*
+  - job_name: ray
+    static_configs:
+      - labels:
+        job: ray
+        __path__: /tmp/ray/session_latest/logs/serve/*.*
 ```
 
-The relevant part for Ray is the `static_configs` field, where we have indicated the location of our log files with `__path__`.
+The relevant part for Ray Serve is the `static_configs` field, where we have indicated the location of our log files with `__path__`.
 The expression `*.*` will match all files, but not directories, which cause an error with Promtail.
```

Comment:
What does this sentence mean? Is it saying that directories in general cause an error with Promtail? Or is it saying that the expression `*.*` matches all files (but not directories) that cause an error with Promtail?

Reply:
Hmm I'm not actually sure but I'd interpret it as "this matches all files, but doesn't match any directories, which causes an error with Promtail."

I think it was written by @simon-mo or @architkulkarni ?

Reply:
I wrote this a while ago, I believe it means that directories cause an error with promtail, so use `*.*` to not match directories

Reply:
yup @architkulkarni is correct

Reply:
Thanks, I revised the wording.